hack/test_e2e.sh

```diff
@@ -465,9 +488,16 @@ function set_e2e_dir {
     pushd "${0%/*}" > /dev/null || exit
         export E2E_DIR="$(pwd)/temp/e2e"
         export E2E_ARTIFACTS="${E2E_DIR}"/artifacts
+        E2E_BIN="${E2E_DIR}"/bin
         mkdir -p "${E2E_DIR}"
         mkdir -p "${E2E_ARTIFACTS}"
-    popd > /dev/null || exit
+        mkdir -p "${E2E_BIN}"
+   popd > /dev/null || exit
```

Comment:
removed space? Usually popd is aligned with pushd

Reply:
Thanks,